cmd/instance/instance_create.go

```diff
@@ -139,8 +139,9 @@ If you wish to use a custom format, the available fields are:
 			}
 		}
 
+		// Set public ipv4 if provided
 		if publicip != "" {
-			config.PublicIPRequired = publicip
+			config.ReservedIPv4 = publicip
```

Comment:
PublicIPRequired is either true o false and it is random and at no cost. It differs from the reserved IP which costs 3 dollars a month.

Reply:
Yes we set that to "true" by default - https://github.com/civo/civogo/blob/master/instance.go#L190